cmd/program/program.go

```diff
@@ -571,4 +669,40 @@ func (p *Project) CreateFileWithInjection(pathToCreate string, projectPath strin
 	}
 
 	return nil
-}
\ No newline at end of file
+}
+
+func (p *Project) CreateTemplateRoutes() {
+	placeHolder := ""
+	if p.AdvancedOptions["AddHTMXTempl"] {
+		placeHolder += string(p.FrameworkMap[p.ProjectType].templater.HtmxTemplRoutes())
+	}
+
+	phTmpl, err := template.New("routes").Parse(placeHolder)
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
if placehodler does not get re-written, is it going to log fatal?

Reply:
When running through it with the flag being false, it doesn't not log fatal. The Parse is just taking in a string so I think it'll be fine no matter what the string is, right?